README.md

```diff
@@ -214,6 +213,9 @@ Build and test orchestration is performed by [Jenkins][21].
 - A listing of connected servers for testing, building and benchmarking
   can be found at: <https://ci.nodejs.org/computer/>
 
+The Build WG will keep build configuration required for a release line for 6
+months after the release goes End-of-Life, in case further build or test runs
+are required. After that the configuration will be removed.
```

Comment:
s/will/may, mayhaps?

Reply:
I'd say will is more accurate. Of course it doesn't specify how soon it will be removed, so there's no rush.

But if anyone reads this, I'd like them to think that this is definitely going away, rather than maybe going away.